jupyterhub/roles.py

```diff
@@ -164,13 +164,12 @@ def expand_roles_to_scopes(orm_object):
     if not isinstance(orm_object, orm.Base):
         raise TypeError(f"Only orm objects allowed, got {orm_object}")
 
-    pass_roles = orm_object.roles
+    pass_roles = []
+    pass_roles.extend(orm_object.roles)
```

Comment:
This is the actual bug fix, where a new list is created and initialized as a copy instead of creating a new variable pointing to the same list. Right?

Reply:
Yes :)